### DISCUSSION TITLE
How to add multiple pages to my site

### DISCUSSION THREAD

**Discussion by BorealBlizzard** [community/community#23746] [Other Feature Feedback, Questions, & Ideas]
<p>Im just learning how to use html and some basic CSS and I was wondering how I could add multiple pages without using Jekyll or any third party help. Is there anyway to do this using only html and the repo?</p>
<p>EDIT: What I ment was how do I link to another HTML file through my index.html file, I thought I did it right and it works when I try it offline but Im not sure how to get it to work through the repo copy. </p>
<p>SITE: <a href="https://borealblizzard.github.io/" rel="nofollow noopener">https://borealblizzard.github.io/</a></p>
<p>REPO: <a href="https://github.com/BorealBlizzard/BorealBlizzard.github.io" rel="nofollow noopener">https://github.com/BorealBlizzard/BorealBlizzard.github.io</a></p>

**Comment by iordic** [comment, marked as answer]
<p>You are using absolute path and you have to use relative path instead.</p>
<p>In the “a” tag replace:</p>
<pre><code>&lt;a href="https://BorealBlizzard.github.io/page1.html/"&gt;Page 2&lt;/a&gt;
</code></pre>
<p>To:</p>
<pre><code>&lt;a href="page1.html"&gt;Page 2&lt;/a&gt;
</code></pre>
<p>Same at <em>page1.html</em> file. :smiley:</p>

**Reply by WildSeal25** [reply]
This was very helpful

**Comment by BorealBlizzard** [comment]
<p>Thank You. Im supriesed I havn’t seen that anywhere.</p>

**Comment by SethClydesdale** [comment]
<p>Hi,</p>
<p>I just wanted to add a bit more information in regards to linking to pages in other directories. As iordic stated you can link to other pages in the current directory like so…</p>
<pre><code>&lt;a href="help/"&gt;Help&lt;/a&gt;
</code></pre>
<p>This will link to the page “/my-project/help/”, if you’re viewing the page on your local file system you’ll have to append index.html. (You can use a JS snippet <a href="https://raw.githubusercontent.com/SethClydesdale/sethclydesdale.github.io/master/resources/javascript.js" rel="nofollow noopener">like this</a> to automatically append it, too) Once we’re down one level in the directory you can do the following to link to pages up one level (or however many levels)</p>
<pre><code>&lt;a href="../"&gt;Home&lt;/a&gt;
</code></pre>
<p>This link will take you back to the root-level of your project : “/my-project/,” from “/my-project/help/.” The dots “…” refer to a parent directory, you can chain them together (i.e. <code>href="../../../"</code>) depending how deep your page is. If you want to link to the current page, you can create a link like this :</p>
<pre><code>&lt;a href="./"&gt;Home&lt;/a&gt;
</code></pre>
<p>The single dot “.” refers to the current directory.</p>
<p>If you want some more examples, you can look over the code for <a href="https://github.com/SethClydesdale/genki-study-resources" rel="nofollow noopener">one of my projects</a>. It doesn’t use Jekyll, just static HTML pages so it can be used online or offline without setting anything up.</p>

**Comment by ghost** [comment]
<p>I am unable to link the pages on my site. it shows 404 page not found<br>
please help me</p>

**Reply by JL-Freeza** [reply]
Yeah Me too

**Comment by real-nickel** [comment]
<p>It Says 404 Not Found</p>

**Comment by real-nickel** [comment]
<p>Works now</p>
<pre><code>&lt;a href="help.html"&gt;help&lt;/a&gt;
</code></pre>

**Comment by Rpascar1** [comment]
<p>This was a great help and explanation to my problem thank you </p>

**Comment by 0xmmalik** [comment]
<p>How would I do it using markdown instead of HTML? Would I just refer it to</p>
<pre><code>page-name.md
</code></pre>
<p>? Or is there something different I have to do?</p>

**Comment by olive89aja** [comment]
<p>Comme collect your drinks when you want. Premium advice right there.</p>
<blockquote>
<p><a class="mention" href="/iordic">@iordic</a> wrote:</p>
</blockquote>
<blockquote>
<p>You are using absolute path and you have to use relative path instead.</p>
</blockquote>
<blockquote>
<p>In the “a” tag replace:</p>
</blockquote>
<blockquote>
<p><a href="https://BorealBlizzard.github.io/page1.html/" rel="nofollow noopener">Page 2</a></p>
</blockquote>
<blockquote>
<p>To:</p>
</blockquote>
<blockquote>
<p><a>Page 2</a></p>
</blockquote>
<blockquote>
<p>Same at <em>page1.html</em> file. :smiley:</p>
</blockquote>

**Comment by Maxicrap** [comment]
<p>THANK YOU </p>

**Comment by lackgagese** [comment]
<p>You can give the feedback on github <a href="https://todaysambadlottery.in/" rel="nofollow noopener">here</a> 's the link <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">https://support.github.com/contact/feedback?contact[category]=actions</a></p>

**Comment by ihsane22** [comment]
how can i upload a multiple page website in github . My website needs 2 pages the home.html and index.html  . But only index.html is shown in my online public website . How can i make the home page show first before the index one ? Please answer me !

**Reply by Rpascar1** [reply]
TBH the question is not 100% clear.

The index page is the home page. you may need to just rename the home page appropriately throughout the code.

Not sure what you mean website needs 2 pages; a repo will contain all pages and then a link within your site code will redirect to subsequent pages. You can’t load 2 pages simultaneously in a single page browser.

Maybe more details can help clarify.

Will you please include a link to the repo like the original poster did?

**Comment by kmadaminjanova0508** [comment]
you can use relative path instead of absolute ;)
